javascript-data-model/order-history.js

```diff
@@ -0,0 +1,33 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4, 2020',
+    totalPrice: '$34.00',
+    expectedDelivery: 'August 8, 2020',
+    itemCategory: 'book',
+    returnWindowClosed: 'September 7, 2020'
+
+  },
```

Comment:
Where's the data about the items in the order? You need to save all the data necessary to generate the output in the screenshot from the instructions.